frontend/integration-tests/tests/kubevirt/vm.wizard.scenario.ts

```diff
@@ -0,0 +1,101 @@
+import { execSync } from 'child_process';
+import { browser, by, ExpectedConditions as until } from 'protractor';
+import { appHost, testName } from '../../protractor.conf';
+import { resourceRowsPresent, filterForName, deleteRow, isLoaded, createItemButton } from '../../views/crud.view';
+import * as vm from '../../views/kubevirt/vm.view';
+
+
+describe('Kubevirt create VM using wizard', () => {
+  const namespace = 'default';
+  const provisionSource = 'PXE';
+  const operatingSystem = 'fedora28';
+  const flavor = 'small';
+  const workloadProfile = 'generic';
+  const networkDefinitionName = `${testName}-ovs-net-1`;
+  const pxeInterface = 'eth1';
+
+  const testNAD = {
+    apiVersion: 'k8s.cni.cncf.io/v1',
+    kind: 'NetworkAttachmentDefinition',
+    metadata: {
+      name: networkDefinitionName,
+    },
+    spec: {
+      config: '{ "cniVersion": "0.3.1", "type": "ovs", "bridge": "br0" }',
+    },
+  };
+
+  beforeAll(async() => {
+    execSync(`echo '${JSON.stringify(testNAD)}' | kubectl create -f -`);
```

Comment:
How about move `nad` to a separate file so it can be a common step and other tests can import it.

Reply:
I totally agree with having some common resources or utilities even may be useful for us, however there is no such convention in the rest of the integration tests repository.
As creating such is rather big change, I think introducing it would be better in a new PR.